packages/mui-base/src/AutocompleteUnstyled/useAutocomplete.js

```diff
@@ -846,6 +846,16 @@ export default function useAutocomplete(props) {
             });
           }
           break;
+        case 'Delete':
+          if (multiple && !readOnly && inputValue === '' && value.length > 0) {
+            const index = focusedTag === -1 ? value.length - 1 : focusedTag;
+            const newValue = value.slice();
+            newValue.splice(index, 1);
+            handleValue(event, newValue, 'removeOption', {
+              option: value[index],
+            });
+          }
+          break;
```

Comment:
Looks like the code is duplicated. Could you add the `case 'Delete'` to the previous case block?

Reply:
Awesome, thank you! I will do that asap.